ArduPlane/tiltrotor.cpp

```diff
@@ -460,6 +525,8 @@ void Tiltrotor::tilt_compensate_angle(float *thrust, uint8_t num_motors, float n
             thrust[i] *= scale;
         }
     }
+
+    _max_tilting_motor_thrust_demand = largest_tilted;
```

Comment:
strictly this should be:
```suggestion
    _max_tilting_motor_thrust_demand = MIN(largest_tilted, 0.0);
```
To take account of the scale back on the line above.

But it makes no difference to the function of this patch.

I guess you could make it a bool `thrust_limited` that would move the threshold to 100% rather than 95% tho.

Reply:
I'd rather have the maximum before limiting.